exercises/sum-of-multiples/description.md

```diff
@@ -4,7 +4,4 @@ but not including that number.
 If we list all the natural numbers up to but not including 20 that are
 multiples of either 3 or 5, we get 3, 5, 6 and 9, 10, 12, 15, and 18.
 
-The sum of these multiples is 78.
-
-Given a number, find the sum of the multiples of a given set of numbers,
-up to but not including that number.
+The sum of these multiples is 78.
```

Comment:
I would prefer that the trailing newline of line 7 does not get deleted, so that line 7 does not show as a changed line, please. Only lines 8-10 should get deleted please.

Reply:
Fixed & squashed.